### DISCUSSION TITLE
Figuring out the ip address from the hellp packet

### DISCUSSION THREAD

**Discussion by choejms** [ANLAB-KAIST/KENSv3#239] [Project Q&A 3. Internet Layer (IP/Routing)]
Hi, when a router sends out hello packets through multicast and if another router receives it, is there a way to know we know which interface the packet was received through?

**Comment by Pusnow** [comment]
You can check the source IP address is on the same subnet as the port's IP address.

**Reply by choejms** [reply]
thank you so much for the quick response!
